redisinsight/api/src/modules/database/database.service.spec.ts

```diff
@@ -96,6 +100,14 @@ describe('DatabaseService', () => {
           provide: DatabaseAnalytics,
           useFactory: mockDatabaseAnalytics,
         },
+        {
+          provide: TagService,
+          useValue: {
+            getOrCreateByKeyValuePairs: jest.fn(),
+            delete: jest.fn(),
+            cleanupUnusedTags: jest.fn(),
+          },
+        },
```

Comment:
let's create `tags.ts` file in the `__mocks__` directory and put all services and mocked models/dtos there.

Reply:
or you can use `database.ts` file for it if you want